x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -180,44 +179,53 @@ function showResults(resp, action) {
 
 export async function cloneJob(jobId) {
   try {
-    const job = await loadFullJob(jobId);
-    if (job.custom_settings && job.custom_settings.created_by) {
+    const [cloneableJob, originalJob] = await Promise.all([
+      loadFullJob(jobId, true),
```

Comment:
`loadFullJob` shouldn't be used here anymore as it adds job and datafeed stats to the job object.
The job and datafeed should be loaded directly from es. Ideally they would also not be combined and we would have a separate `datafeed` object in `tempJobCloningObjects`. but if that has unexpected side effects we can continue to combine them into a `CombinedJob`.

Reply:
I have updated added a new method called `loadJobForExport` which will only get the `CombinedJob` with datafeed and the job config with `exclude_generated: true`. I also tried fetching them separately but encountered some side effects with the wizard behaving unpredictably, but if that's still the preferred method I can revert the change to do that instead.

Reply:
Updated this to `getJobForCloning` which will return them separately with `exclude_generated` to always be true.